app/src/main/java/com/concordium/wallet/ui/cis2/SendTokenReceiptActivity.kt

```diff
@@ -94,8 +93,9 @@ class SendTokenReceiptActivity : BaseActivity() {
     }
 
     private fun onFinish() {
-        val intent = Intent(this, AccountDetailsActivity::class.java).apply {
+        val intent = Intent(this, SendTokenActivity::class.java).apply {
```

Comment:
When I try this out, and send CCD from the account details activity page, then I start the send token flow.

After I click finished, I’m routed directly in this flow SendTokenActivity -> AccountDetailsActivity (due to `putExtra(SendTokenActivity.FLOW_STATE, true)`)

I do however get a small “blink” of the SendTokenActivity page - can’t we avoid this since it doesn’t look nice?

Reply:
that is the problem with activities, we are not using fragments where I could simply change the back stack etc. 
other solutions to go back 2 activities are super hacky and I prefer not to use them. 
I'll push an alternative version please verify if it works better.